README.md

```diff
@@ -19,11 +18,10 @@ The live site can be viewed [here]().
 * Organizer can create an event
 * User can create a group
 * Organizer can set Category for group
-
+* Organizer can set additional event info
```

Comment:
What does that mean?

Reply:
Right.  Wasn't sure with this one but I guess it's not really a feature.  Has been deleted.